cosmos/keybase.go

```diff
@@ -1,17 +1,24 @@
 package cosmos
 
 import (
+	"sync"
+
 	clientkey "github.com/cosmos/cosmos-sdk/client/keys"
 	"github.com/cosmos/cosmos-sdk/crypto/keys"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/hd"
 	"github.com/cosmos/cosmos-sdk/crypto/keys/keyerror"
+	"github.com/cosmos/cosmos-sdk/types"
 	bip39 "github.com/cosmos/go-bip39"
+	"github.com/tendermint/tendermint/crypto"
 )
 
 const mnemonicEntropySize = 256
 
 // Keybase is a standard cosmos keybase.
 type Keybase struct {
 	keys.Keybase
```

Comment:
let's not inherit this if we redefine all the functions and let's make it private too
```suggestion
	keybase keys.Keybase
```

Reply:
agree with antho. Otherwise we could "forget" to implement the lock on a function..

Reply:
I left keybase composed because someone might still want to use it. I can make it private.